src/libServer/LookupServer.cpp

```diff
@@ -87,6 +89,7 @@ LookupServer::LookupServer(Mediator& mediator,
       jsonrpc::Procedure("GetCurrentDSEpoch", jsonrpc::PARAMS_BY_POSITION,
                          jsonrpc::JSON_STRING, NULL),
       &Server::GetCurrentDSEpochI);
+
```

Comment:
Much whitespace makes fewer lines fit on the screen.  I understand the preference, but we first should agree about this style choice, then make the change.

Lack of this space is not a bug, so need not be fixed.  Also tagging @CSideSteve, @n-hutton, @art-gor, @KaustubhShamshery for an opinion about blank lines.

Another question is whether to keep style cleanup PRs separate from the functionality implementation.

Reply:
IMHU, an extra line after a code-block makes the code more readable